Sources/WeatherSDK/AuthSchemeResolver.swift

```diff
@@ -1,13 +1,16 @@
 // Code generated by smithy-swift-codegen. DO NOT EDIT!
 
-import ClientRuntime
```

Comment:
Many WeatherSDK files changed below due to codegen changes.

You can look to see how service clients are affected, or you can skip past WeatherSDK entirely.